ozark/pom.xml

```diff
@@ -79,13 +120,37 @@
                     </execution>
                 </executions>
             </plugin>
+            <!--
+             This plugin is reponsible for packaging artifacts
+             as OSGi bundles.  Please refer to
+             http://felix.apache.org/site/apache-felix-maven-bundle-plugin-bnd.html
+             for more information about how to use this plugin.
+           -->
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
```

Comment:
Maven warnings generally messy if you know what to do. Version information comes from the parent "jvnet-parent" so it is not necessary to duplicate it. No one (other specs and impls) worried about that.